docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md

```diff
@@ -0,0 +1,113 @@
+
+## Begin to access Apache APISIX Prometheus Metrics
+
+### Step1: Install APISIX Ingress Controller
+
+First, we deploy Apache APISIX, ETCD, and APISIX Ingress Controller to a local Kubernetes cluster via Helm.
+
+```sh
+helm repo add apisix https://charts.apiseven.com
+helm repo update
+kubectl create namespace ingress-apisix
+helm install apisix apisix/apisix --namespace ingress-apisix \
+ --set ingress-controller.enabled=true
+```
+
+After installation, please wait until all services are up and running. Specific status confirmation can be checked with the following command.
+
+```sh
+kubectl get all -n ingress-apisix
+```
+
+### Step 2: Enable Prometheus Plugin
+
+If you need to monitor Apache APISIX at the same time, you can create the following ApisixClusterConfig resource.
+
+```yaml
+apiVersion: apisix.apache.org/v2beta3
+kind: ApisixClusterConfig
+metadata:
+  name: default
+spec:
+  monitoring:
+    prometheus:
+      enable: true
+```
+
+### Step 3: Enable `public-api` Plugin
+
+This is a basic routing setup, please note that further configuration should be done based on your local backend service information. The main solution concept is to use the `public-api` plugin to protect the routes exposed by **Prometheus**. For a more detailed configuration, you can refer to the [example](https://apisix.apache.org/zh/docs/apisix/plugins/public-api/#example) section of the `public-api` plugin.
+
+```yaml
+apiVersion: apisix.apache.org/v2beta3
+kind: ApisixRoute
+metadata:
+  name: prometheus-route
+spec:
+  http:
+  - name: public-api
+    match:
+      hosts:
+      - test.prometheus.org
+      paths:
+      - /apisix/prometheus/metrics
+    backends:
+    ## Please notice that there must be your actual "serviceName" and "servicePort"
+    - serviceName: apisix-test-prometheus
+      servicePort: 9180
+    plugins:
+    - name: public-api
+      enable: true
+```
+
+### Step 4: Collect the Metrics
+
+Now you can then get the indicator parameters by requesting command access.
+
+```sh
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9180/apisix/admin/routes -H 'X-API-Key: edd1c9f034335f136f87ad84b625c8f1'
+
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9080/headers -H 'Host: test.prometheus.org'
+```
+
+## Conclusion
+
+This article describes the steps on how to use the `public-api` plugin to protect **Prometheus** and monitor the APISIX Ingress Controller. Currently, only some basic configurations are included, if you want to see some of the metrics displayed with **Grafana**, please refer to this [link](https://apisix.apache.org/zh/blog/2021/12/13/monitor-apisix-ingress-controller-with-prometheus/#). We will continue to polish and upgrade, add more metrics and integrate data surface APISIX metrics to bring you a better monitoring experience.
```

Comment:
no, this is for monitoring APISIX not APISIX Ingress controller

Reply:
Got it!